BLDR-Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM alpine:3.6
+MAINTAINER The Habitat Maintainers <humans@habitat.sh>
+
+ARG APP_HOSTNAME=localhost
+ARG GITHUB_API_URL=https://api.github.com
+ARG GITHUB_WEB_URL=https://github.com
+ARG GITHUB_CLIENT_ID
+ARG GITHUB_CLIENT_SECRET
+ARG GITHUB_ADMIN_TEAM
+ARG GITHUB_WORKER_TEAM
+ARG WORKER_AUTH_TOKEN
+
+ENV HAB_DEPOT_CHANNEL unstable
+ENV RUST_LOG info
+
+COPY support/builder/hab-entrypoint.sh /usr/local/bin/hab-entrypoint.sh
+COPY terraform/scripts/install_base_packages.sh /tmp/install_base_packages.sh
+COPY terraform/scripts/foundation.sh /tmp/foundation.sh
+
+# add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
+RUN addgroup -S hab && adduser -S -G hab hab
+
+RUN apk update
```

Comment:
We could set up a virtual package group [like this](https://github.com/bscott/habitat-docker/blob/e4a7c9cd5175b0fa85701602e46ce164e5e00eb6/Dockerfile#L5-L9) so that we can delete them after we are done with them [like this](https://github.com/bscott/habitat-docker/blob/e4a7c9cd5175b0fa85701602e46ce164e5e00eb6/Dockerfile#L15).

The only catch is that those packages would still exist as lower level image layers, which is why it might be useful to run the `apk update` through to `rm -Rf hab_builder_bootstrap*` as a single `RUN` command.